README.md

```diff
@@ -78,54 +79,9 @@ Yes! Want to help build Kedro? Check out our [guide to contributing to Kedro](ht
 
 ## Where can I learn more?
 
-There is a growing community around Kedro. Have a look at the [Kedro FAQs](https://docs.kedro.org/en/stable/faq/faq.html#how-can-i-find-out-more-about-kedro) to find projects using Kedro and links to articles, podcasts and talks.
-
-## Who likes Kedro?
-
```

Comment:
I think the only thing that's now missing from the README.md is how do contributors know that they can add themselves to this list on `awesome-kedro`?

Reply:
Good point, I'll add a sentence.